content/docs/next-release/operator.md

```diff
@@ -0,0 +1,645 @@
+
+
+```yaml
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: cassandra-without-create-schema
+spec:
+  strategy: allInOne
+  storage:
+    type: cassandra
+    cassandraCreateSchema:
+      enabled: false # <1>
+```
+<1> Defaults to `true`
+
+Further aspects of the batch job can be configured as well. An example with all the possible options is shown below:
+
+
+```yaml
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: cassandra-with-create-schema
+spec:
+  strategy: allInOne # <1>
+  storage:
+    type: cassandra
+    options: # <2>
+      cassandra:
+        servers: cassandra
+        keyspace: jaeger_v1_datacenter3
+    cassandraCreateSchema: # <3>
+      datacenter: "datacenter3"
+      mode: "test"
+```
+<1> The same works for `production` and `streaming`.
+
+<2> These options are for the regular Jaeger components, like `collector` and `query`.
+
+<3> The options for the `create-schema` job.
+
+NOTE: The default create-schema job uses `MODE=prod`, which implies a replication factor of `2`, using `NetworkTopologyStrategy` as the class, effectively meaning that at least 3 nodes are required in the Cassandra cluster. If a `SimpleStrategy` is desired, set the mode to `test`, which then sets the replication factor of `1`. Refer to the [create-schema script](https://github.com/jaegertracing/jaeger/blob/master/plugin/storage/cassandra/schema/create.sh) for more details.
+
+
+### Elasticesearch storage
+
+Under some circumstances, the Jaeger Operator can make use of the [Elasticsearch Operator](https://github.com/openshift/elasticsearch-operator) to provision a suitable Elasticsearch cluster.
+
+IMPORTANT: This feature is experimental and has been tested only on OpenShift clusters. Elasticsearch also requires the memory setting to be configured like `minishift ssh -- 'sudo sysctl -w vm.max_map_count=262144'`. Spark dependencies are not supported with this feature [Issue #294](https://github.com/jaegertracing/jaeger-operator/issues/294).
+
+When there are no `es.server-urls` options as part of a Jaeger `production` instance and `elasticsearch` is set as the storage type, the Jaeger Operator creates an Elasticsearch cluster via the Elasticsearch Operator by creating a Custom Resource based on the configuration provided in storage section. The Elasticsearch cluster is meant to be dedicated for a single Jaeger instance.
+
+The self-provision of an Elasticsearch cluster can be disabled by setting the flag `--es-provision` to `false`. The default value is `auto`, which will make the Jaeger Operator query the Kubernetes for its ability to handle a `Elasticsearch` custom resource. This is usually set by the Elasticsearch Operator during its installation process, so, if the Elasticsearch Operator is expected to run *after* the Jaeger Operator, the flag can be set to `true`.
```

Comment:
`s/query the Kubernetes for/query Kubernetes for/`, or
`s/query the Kubernetes for/query the Kubernetes cluster for/`